Userland/test-crypto.cpp

```diff
@@ -1558,6 +1596,50 @@ void tls_test_client_hello()
     loop.exec();
 }
 
+int adler32_tests()
+{
+    auto do_test = [](ReadonlyBytes input, u32 expected_result) {
+        I_TEST();
```

Comment:
Test what?
Please at the very least say that it's testing Adler32: `I_TEST((Adler32));`